extensions/wikia/AdEngine/js/slot/service/actionHandler.js

```diff
@@ -3,8 +3,9 @@ define('ext.wikia.adEngine.slot.service.actionHandler',  [
 	'ext.wikia.adEngine.messageListener',
 	'ext.wikia.adEngine.slotTweaker',
 	'ext.wikia.adEngine.slot.service.viewabilityHandler',
+	'wikia.abTest',
 	'wikia.log'
-], function (messageListener, slotTweaker, viewabilityHandler, log) {
+], function (messageListener, slotTweaker, viewabilityHandler, abTest, log) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Function has 5 parameters which is greater than 4 authorized. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)